scripts/test/updateDataWidgets.js

```diff
@@ -0,0 +1,31 @@
+const { exec } = require("child_process");
+const { join } = require("path");
+const rCopy = require("recursive-copy");
+const { promisify } = require("util");
+
+main().catch(e => {
+    console.error(e);
+    process.exit(-1);
+});
+
+async function main() {
+    await copyLatestDataWidgets();
+}
```

Comment:
We should add removal of the existing folders on the test project

Reply:
Thanks! I didn't remove the theme folder because, in the atlas update script, we are already doing that.

Done! ✅